hera_qm/omnical_metrics.py

```diff
@@ -166,10 +166,12 @@ def load_firstcal_gains(fc_file):
     uvf = UVCal()
     uvf.read_calfits(fc_file)
     freqs = uvf.freq_array.squeeze()
-    firstcal_delays = np.moveaxis(uvf.delay_array, 2, 3)[:, 0, :, :, :]
-    firstcal_gains = np.array(map(lambda x: np.exp(-2j * np.pi * freqs.reshape(1, -1, 1) * x), firstcal_delays))
+    fc_gains = np.moveaxis(uvf.gain_array, 2, 3)[:, 0, :, :, :]
+    d_nu = np.mean(freqs[1:]-freqs[:-1])
+    d_phi = np.abs(np.mean(np.angle(fc_gains)[:, :, 1:, :] - np.angle(fc_gains)[:, :, :-1, :], axis=2))
+    fc_delays = (d_phi / d_nu)/(2*np.pi)
```

Comment:
This code is now repeated in firstcal_metrics and omnical_metrics. We should move it into a utility function, but it's fine if we wait until after this PR.